tikv-client/src/main/java/com/pingcap/tikv/txn/LockResolverClientV2.java

```diff
@@ -46,23 +46,21 @@
 import org.tikv.kvproto.TikvGrpc.TikvStub;
 
 /** Before v3.0.5 TiDB uses the ttl on secondary lock. */
-
-// LockResolver resolves locks and also caches resolved txn status.
 public class LockResolverClientV2 extends AbstractRegionStoreClient
     implements AbstractLockResolverClient {
-  // ResolvedCacheSize is max number of cached txn status.
-  private static final long RESOLVED_TXN_CACHE_SIZE = 2048;
-
   private static final Logger logger = LoggerFactory.getLogger(LockResolverClientV2.class);
 
   private final ReadWriteLock readWriteLock;
-  // Note: Because the internal of long is same as unsigned_long
-  // and Txn id are never changed. Be careful to compare between two tso
-  // the `resolved` mapping is as {@code Map<TxnId, TxnStatus>}
-  // TxnStatus represents a txn's final status. It should be Commit or Rollback.
-  // if TxnStatus > 0, means the commit ts, otherwise abort
+
+  /**
+   * Note: Because the internal of long is same as unsigned_long and Txn id are never changed. Be
```

Comment:
I think we should use // instead of /**/ for these comments

Reply:
![image](https://user-images.githubusercontent.com/5574887/83221282-f7f72100-a1a7-11ea-80d5-451d883022e2.png)

Reply:
![image](https://user-images.githubusercontent.com/5574887/83221337-1eb55780-a1a8-11ea-9c5c-eaedf63ced73.png)